src/pages/settings/Security/TwoFactorAuth/StepWrapper/StepWrapperPropTypes.tsx

```diff
@@ -1,28 +1,29 @@
-import PropTypes from 'prop-types';
-
-export default {
+type StepWrapperPropTypes = {
```

Comment:
rename this to `StepWrapperProps` and move this type to `StepWrapper.tsx` we should create seperate type files only when there are platform specific files